codalab/apps/chahub/models.py

```diff
@@ -119,5 +155,19 @@ def save(self, force_to_chahub=False, *args, **kwargs):
 
                     # We save at the beginning, but then again at the end to save our new chahub timestamp and such
                     super(ChaHubSaveMixin, self).save(force_update=True)
-            else:
-                logger.info("ChaHub :: Model failed validation")
+            elif not is_valid and self.chahub_needs_retry:
+                # This is NOT valid but also marked as need retry, unmark need retry until this is
+                # valid again
+                self.chahub_needs_retry = False
+                super(ChaHubSaveMixin, self).save(force_update=True)
+
+    def delete(self, real_delete=False, **kwargs):
+        if real_delete:
+            super(ChaHubSaveMixin, self).delete(**kwargs)
+        else:
+            # So this can be sent to ChaHub, with retries and such, don't actually delete
+            self.deleted = True
+            self.save()
```

Comment:
we want to _not_ send to chahub here, right?

Reply:
I believe we do? can hop in a chat to talk about this stuff if that's quicker

Reply:
don't need to send to chahub here since the `delete_from_chahub` below is sending the same information again

Reply:
Adding a kwarg to save so we can `send_to_chahub=False`

Reply:
✔️